march_hardware_interface/config/march-iii/joint_limits.yaml

```diff
@@ -0,0 +1,79 @@
+joint_limits:
+  left_hip:
+    has_position_limits: true
+    min_position: -0.26
+    max_position: 1.83259571
+    has_velocity_limits: true
+    max_velocity: 0.2
+
+    has_soft_limits: true
+    soft_lower_limit: -0.2
+    soft_upper_limit: 1.75
+    k_position: 10.0
+    k_velocity: 10.0
+
+  right_hip:
+    has_position_limits: true
+    min_position: -0.26
+    max_position: 1.83259571
+    has_velocity_limits: true
+    max_velocity: 0.2
+
+    has_soft_limits: true
+    soft_lower_limit: -0.2
+    soft_upper_limit: 1.75
+    k_position: 10.0
+    k_velocity: 10.0
+
+  left_knee:
+    has_position_limits: true
+    min_position: -0.2
+    max_position: 2
+    has_velocity_limits: true
+    max_velocity: 0.2
+
+    has_soft_limits: true
+    soft_lower_limit: -0.25
+    soft_upper_limit: 1.75
+    k_position: 10.0
+    k_velocity: 10.0
+
+  right_knee:
+    has_position_limits: true
+    min_position: -0.2
+    max_position: 2
+    has_velocity_limits: true
+    max_velocity: 0.2
+
+    has_soft_limits: true
+    soft_lower_limit: -0.25
+    soft_upper_limit: 1.75
+    k_position: 10.0
+    k_velocity: 10.0
+
+  right_ankle:
+    has_position_limits: true
+    min_position: -0.349
+    max_position: 0.265
+    has_velocity_limits: true
+    max_velocity: 0.2
+
+    has_soft_limits: true
+    soft_lower_limit: -0.32
+    soft_upper_limit: 0.25
+    k_position: 10.0
+    k_velocity: 10.0
+
+  left_ankle:
+    has_position_limits: true
+    min_position: -0.366
+    max_position: 0.296
+    has_velocity_limits: true
+    max_velocity: 0.2
+
+    has_soft_limits: true
+    soft_lower_limit: -0.35
+    soft_upper_limit: 0.293
+    k_position: 10.0
+    k_velocity: 10.0
+
```

Comment:
How are these values determined? Is that documented somewhere? Or will they be revised in the future?

Reply:
Completely made up, will be refactored and tweaked after march_hardware_builder is merged.

What about it would you like to see documented and why?

Reply:
If they are made up, I think this must be mentioned somewhere. Since if we forget to adjust it, M5 will think that we gave these variables these values for a reason.

Reply:
If we had a reason, I think this must be documented with arguments on the Confluence, since it is an important decision that M5 might want to revise.

Reply:
For example it is important why M3 choose 5 degrees for their software limits, because if we want to change that we should have a reason that is better then their reasoning.

Reply:
Will be done with the final values